docker-compose.yml

```diff
@@ -5,10 +5,11 @@ services:
      IP: ${REDIS_CLUSTER_IP}
      SENTINEL: ${REDIS_USE_SENTINEL}
      STANDALONE: ${REDIS_USE_STANDALONE}
+     TLS: ${REDIS_USE_TLS}
     build:
       context: .
       args:
-        redis_version: '6.0.1'
+        redis_version: '6.0.8'
```

Comment:
Please revert back this one. I will do a race and add all new versions in the 6.0.x track later as there is more then one place to add configuration and documentation when adding new releases.